### DISCUSSION TITLE
Impossible to change sailing mode

### DISCUSSION THREAD

**Discussion by ange335** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#436] [General]
AlDente works now fine at my Device. The only problem I´m facing is, that I can not change sailing mode. I set my charging to 80% max so I can handle 2 days heavy work load out of office without charging. I browsed through the settings and found, that my sailing is still at 75% and I´m not able to change it (I tried restart of the app, reboot the Mac, toggle on/off sailing mode). 

When I change the 5% sailing difference to let's say 20% the sail through value changes to 60% even I had set 80% as my charging limit... when I enter 5% again the system changes to 75% sail through value. Only when I enter 0 than the value changes to 80%, when I enter 9 the charging through value is 71% - so there must be a error in calculations.  Strange behaviour... A bug or just a mistake in interpretation from my side? 

Device: MacBookPro 18,3 (M1 Pro, 14 inch, 2021 )
OS: macOs 12.1 (21C5021h)

<img width="425" alt="Screen 2021-11-03 um 01 09 24" src="https://user-images.githubusercontent.com/78279658/139968416-533ea101-cb71-4d6e-a75f-827de702cc77.png">
<img width="416" alt="Screen 2021-11-03 um 00 55 57" src="https://user-images.githubusercontent.com/78279658/139968409-3b8f6a6f-7a26-43e3-aa72-06b80bdf3bbf.png">
<img width="413" alt="Screen 2021-11-03 um 01 11 20" src="https://user-images.githubusercontent.com/78279658/139968550-f90f9a57-652b-4fcc-b4ef-22fb50cdef9a.png">
<img width="429" alt="Screen 2021-11-03 um 01 11 39" src="https://user-images.githubusercontent.com/78279658/139968555-0017809b-6681-4b6a-b4a4-b778cbd7455a.png">

**Comment by MatthiasKerbl** [comment]
Hi @ange335

You might have misunderstood the functionality of the Sailing Mode.
Please read the following explanation about the Sailing Mode on our blog: https://apphousekitchen.com/feature-explanation-sailing-mode/

If you have still some questions, let me know.

Cheers,
Matthias
